From 988936aaea5d6a5bd475c6d49559384c71dc444d Mon Sep 17 00:00:00 2001
From: Dan Walsh <dwalsh@redhat.com>
Date: Tue, 15 Dec 2015 13:43:12 -0500
Subject: [PATCH 04/13] Change the default mount mode of containers from
 Private to slave

Docker-DCO-1.1-Signed-off-by: Dan Walsh <dwalsh@redhat.com> (github: rhatdan)

Dan Walsh <dwalsh@redhat.com>

Signed-off-by: Dan Walsh <dwalsh@redhat.com>
---
 daemon/execdriver/driver_unix.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/execdriver/driver_unix.go b/daemon/execdriver/driver_unix.go
index 1a72fd1..81c9f77 100644
--- a/daemon/execdriver/driver_unix.go
+++ b/daemon/execdriver/driver_unix.go
@@ -143,7 +143,7 @@ func InitContainer(c *Command) *configs.Config {
 	container.Readonlyfs = c.ReadonlyRootfs
 	// This can be overridden later by driver during mount setup based
 	// on volume options
-	SetRootPropagation(container, mount.RPRIVATE)
+	SetRootPropagation(container, mount.RSLAVE)
 
 	// check to see if we are running in ramdisk to disable pivot root
 	container.NoPivotRoot = os.Getenv("DOCKER_RAMDISK") != ""
-- 
2.5.0

